public/app/actions/actions.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
```

Comment:
There is a possibility that there can be more modules, likes actions pipeline etc.